src/Prestige.Identity.Infrastructure/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Prestige.Identity.Data.DAL;
+using Prestige.Identity.Data.Domain;
+using System.Threading.Tasks;
+
+namespace Prestige.Identity.Infrastructure.Repositories
+{
+    public class UserRepository : IUserRepository
+    {
+        public UserRepository(PrestigeContext context)
+        {
+            this.Context = context;
+            this.DbSet = this.Context.Set<User>();
+        }
+
+        private PrestigeContext Context { get; }
+
+        private DbSet<User> DbSet { get; }
```

Comment:
This `DbSet` property should be renamed to `Users`. The way it is now, we do not know with which db set we're working exactly.